resource/SQLiteDB.js

```diff
@@ -0,0 +1,49 @@
+// used for testing
+const sqliteDatabase = require('better-sqlite3');
+
+module.exports.createSQLiteDatabase = function(cb) {
+    // Please look at the README for debugging to the console
+    const Testdb = new sqliteDatabase('airlinesTest.db', {memory: true});
+    Testdb.prepare('CREATE TABLE `tbl_airports` (`code` char(3) NOT NULL,`name` varchar(45) NOT NULL,PRIMARY KEY (`code`),UNIQUE (`code`));').run();
```

Comment:
It seems to me it'd probably be better to store these strings outside this function. And the database-structure SQL can surely be loaded from a file? (Even if it differs somewhat from the MySQL version.)

Reply:
Thank you! My code now looks really clean!! I made these changes in #7